docs/release/trg-7/trg-7-09.md

```diff
@@ -0,0 +1,79 @@
+---
+title: TRG 7.09 - Deprecation of Unmaintained Product Repositories
+---
+
+| Status | Created     | Post-History                          |
+|--------|-------------|---------------------------------------|
+| Draft  | 02-Oct-2024 | Initial draft                         |
+
+## Why
+
+Eclipse Tractus-X is an open source project governed by the Eclipse Foundation and its Committers. Beside the role participating in maintaining the code base, within Eclipse Tractus-X the group of committers also acts as the project's security team. They have access to the tab "Security" and may see the vulnerabilities detected by the tooling set up in [TRGs 8](../trg-8/trg-8-01.md). As this group of committers is responsible for the whole project and has limited resources, projects that don't receive any updates for a given time period will be deprecated. This is a "value-for-money" decision.
+
+## Description
+
+Deprecation applies to repositories of type `product`. This information is maintained in the repository metafile (see [TRG 2.05](../trg-2/trg-2-5.md)).Deprecation of a repository means that **the repository is beeing archived**. It's still accessible from within the project, so that it could be revived. Please refer to [this list for examples of archived repositories within Eclipse Tractus-X](https://github.com/eclipse-tractusx/?q=&type=archived&language=&sort=).
+
+Additionally the helm charts and container images are still available and no information about the deprecation is passed to the user.
+
+### Criteria for Deprecation
+
+Committers consider the following criteria to propose deprecation of a repository:
+
+1. The product of the repository has not been released within the *past year's major release cycle* **\***.
+2. The standards used by the product have been changed in a major version within the *past year's major release cycle* **\***.
+
+**\*** The *past year's major release cycle* is the time span between the breaking change releases YY.06 e.g., 24.05 till 24.05.
+
+### Process
+
+To ensure the communities awareness the following process is used at least per major release planning:
+
+```mermaid
+graph TD
+    A[1: Prepare list of unmaintained repositories] --> B[2-1 Inform Eclipse Tractus-X community]
+    B --> C[2-2: Draft critical issue for committer meeting]
+    C --> D[3: 2 week period: Comment on issue for criteria discussion]
+    D --> E[4-1: Committer meeting: Discuss list]
+    E --> F[4-2: Send email to mailing list]
```

Comment:
Isn't this already covered by step 2?

Reply:
My idea was like: make the committers and the community aware of the issue. Wait for a grace period (at least one meeting in between) and then inform community "these are subject to deprecation"

I really want to emphasize, that this also somehow should be aligned best to the release cycle as it makes most sense to make persons prior to planning aware of this. One might also consider to trigger the evaluation always in the `.03` releases.

Reply:
The problem with the dependency to releases (or open plannings) is that that'll extend the grace period to multiple months in the worst case. Do we want that?